controls/radfluidcontentcontrol/fluidcontentcontrol-integration-examples.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Integration Examples
+page_title: Integration Examples
+description: The article contains list with the RadFluidContentControl integration examples with other controls.
```

Comment:
The article contains a list with RadFluidContentControl examples demonstrating how to integrate it with other controls.